runtime-light/stdlib/curl/curl.h

```diff
@@ -4,17 +4,9 @@
 
 #pragma once
 
-#include <cstdint>
+#include "runtime-light/stdlib/curl/curl-context.h"
```

Comment:
I got your idea, you use indirect includes, I'd like to reduce their count as much as possible as it makes code analysis tools go crazy sometime